src/org/jitsi/impl/configuration/ConfigurationServiceImpl.java

```diff
@@ -102,7 +103,8 @@
      * and attempts to do so will simply be ignored.
      * @see #defaultProperties
      */
-    private Map<String, String> immutableDefaultProperties = new HashMap<>();
+    private Map<String, String> immutableDefaultProperties
+        = new ConcurrentHashMap<>();
```

Comment:
That seems unnecessary because immutableDefaultProperties is modified only during the construction of ConfigurationServiceImpl.